.github/workflows/ethernet_init__build.yml

```diff
@@ -17,11 +18,13 @@ jobs:
         uses: actions/checkout@master
         with:
           path: esp_eth_drivers
-      - name: Build ${{ matrix.example }} with IDF-${{ matrix.idf_ver }} for ${{ matrix.idf_target }}
+      - name: Build ${{ matrix.example }} with IDF-${{ matrix.idf_ver }} for ${{ matrix.idf_target }} with ${{ matrix.eth_target }}
         env:
           IDF_TARGET: ${{ matrix.idf_target }}
         shell: bash
         run: |
           . ${IDF_PATH}/export.sh
           cd $GITHUB_WORKSPACE/esp_eth_drivers/ethernet_init/examples/${{ matrix.example }}
+          mv sdkconfig.defaults.${{ matrix.eth_target }} sdkconfig.defaults
+          idf.py reconfigure
```

Comment:
Nice workaround. However, please investigate if it could be done in more "standardized" way. Check https://github.com/espressif/idf-extra-components/tree/master/.github/workflows

Reply:
At first I tried with `idf.py -DSDKCONFIG=sdkconfig.defaults.${{ matrix.eth_target }}` nut it didn't work for some reason. Thank you for the link, I will look into it.

Reply:
Try based on this example: `idf-build-apps build -p apps/iperf --target all --build-dir build_@t_@w --config 'sdkconfig.ci.*='`